### PR TITLE
Live -454: refactored components from JSX to FC

### PR DESCRIPTION
**Pull request by dskamiotis**
## Why are you doing this?

Currently many react components are constructed using JSX.Element with the type definition in the parameter. There is inconsistency across the code base in the way components are constructed.

## Changes

- Used FC<Props> to replace JSX.Elements
- imported FC within React in imports

### CODE REVIEW

File: src/components/shared/leftColumn.tsx [hunk 1]
```diff
@@ -49,8 +49,8 @@ interface LeftColumnProps {
 
 // ----- Component ----- //
 
-const LeftColumn =
-    ({ children, columnContent = null, className = null }: LeftColumnProps): JSX.Element =>
+const LeftColumn: FC<LeftColumnProps> =
+    ({ children, columnContent = null, className = null } ) =>
```
Comment (JamieB-gu) [inline src/components/shared/leftColumn.tsx:53]:
```suggestion
    ({ children, columnContent = null, className = null }) =>
```

**Review by JamieB-gu — COMMENTED** [review]
This is great 🎉!

A few small comments and suggestions.

File: src/components/liveblog/block.tsx [hunk 2]
```diff
@@ -59,23 +59,23 @@ interface TitleProps {
...
     pillar,
     highlighted,
     title,
     children,
     firstPublishedDate,
     lastModifiedDate,
-}: LiveblogBlockProps): JSX.Element => {
-    const relativeFirstPublished = (date: Option<Date>): JSX.Element | null => pipe2(
+}) => {
+    const relativeFirstPublished: FC<Option<Date>> = (date) => pipe2(
```
[the remark this answers is not in the document]
Comment (dskamiotis) [reply src/components/liveblog/block.tsx:78]: I think ill leave it as is makes more sense

File: src/components/media/tags.tsx [hunk 3]
```diff
@@ -39,12 +39,12 @@ interface TagsProps {
...
 }
 
-const Tags = ({ tags, background }: TagsProps): JSX.Element => (
+const Tags: FC<TagsProps> = ({ tags, background }) => (
     <ul css={tagsStyles(background)}>
         {tags.map((tag, index) => {
             return <li key={index}>
                 <a href={tag.webUrl}>
-                    {tag.webTitle}
+                    {tag.webTitle}FRED
```
[the remark this answers is not in the document]
Comment (dskamiotis) [reply src/components/media/tags.tsx:47]: 🤦‍♂️ my testing

File: src/components/liveblog/block.tsx [hunk 4]
```diff
@@ -59,23 +59,23 @@ interface TitleProps {
...
     pillar,
     highlighted,
     title,
     children,
     firstPublishedDate,
     lastModifiedDate,
-}: LiveblogBlockProps): JSX.Element => {
-    const relativeFirstPublished = (date: Option<Date>): JSX.Element | null => pipe2(
+}) => {
+    const relativeFirstPublished = (date: Option<Date>): JSX.Element | null  => pipe2(
```
Comment (JamieB-gu) [inline src/components/liveblog/block.tsx:78]:
```suggestion
    const relativeFirstPublished = (date: Option<Date>): JSX.Element | null => pipe2(
```

**Review by JamieB-gu — APPROVED** [review]

**Pull request merged**